packages/calypso-e2e/src/lib/flows/start-import-flow.ts

```diff
@@ -27,7 +27,7 @@ const selectors = {
 	// Buttons
 	checkUrlButton: 'form.capture__input-wrapper button.action-buttons__next',
 	startBuildingButton: 'div.import__onboarding-page button.action-buttons__next',
-	startImportButton: 'button.select-items-alt__item-button:text("Import your site content")',
+	startImportButton: 'button.import-link__button:text("Import your site content")',
```

Comment:
Using a more common selector would help this test run on both goals and import step.a
```suggestion
	startImportButton: 'button:text("Import your site content")',
```